es8/2017-03/mar-23.md

```diff
@@ -936,7 +936,7 @@ DE: Is this ready for stage 2?
 
 WH: I am happy with it. I did a good review of it with DE. The remaining question for me is how this will work with Array buffers. There are some interesting questions there.
 
-DE: There is an issue on that: issue #23.
+DE: There is an issue on that: [issue #23](https://github.com/tc39/proposal-integer).
```

Comment:
Thanks!

Fixed in https://github.com/rwaldron/tc39-notes/pull/66/commits/184a1709a645f9ae22147cd21539255868de4c3f